tools/cockpit.spec

```diff
@@ -93,6 +93,7 @@ Recommends: %{name}-docker = %{version}-%{release}
 Suggests: %{name}-pcp = %{version}-%{release}
 Suggests: %{name}-kubernetes = %{version}-%{release}
 Suggests: %{name}-selinux = %{version}-%{release}
+Suggests: %{name}-pkgupdates = %{version}-%{release}
```

Comment:
I don't like cockpit-packagekit TBH:
 - This is soon going to interact with dnf/yum/apt specific configuration for setting up automatic updates (as PackageKit doesn't cover that).
 - PackageKit is not solely intended for updates, but also for finding and installing new software. For example, the app install bits that  @mvollmer is working on is also going to use PK. PK is a means to an end, not a functionality description.
 - *If* we are going to add other mechanisms to actually update packages (like calling yum directly instead of PackageKit), these would go into *this* module, not a different one.